Packages/com.chisel.graph/Chisel/Editor/ChiselGraphInstanceEditor.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEditor;
+using UnityEditor.Callbacks;
+using UnityEngine;
+using XNodeEditor;
+
+namespace Chisel.Nodes
+{
+    [CustomEditor(typeof(ChiselGraphInstance))]
+    public class ChiselGraphInstanceEditor : Editor
+    {
+        public override void OnInspectorGUI()
+        {
+            var instance = target as ChiselGraphInstance;
```

Comment:
You should either use the scriptableObject/scriptableProperties in ChiselGraphPropertyEditor.OnGUI(instance)
Or use [DisallowMultipleComponent] on ChiselGraphInstance
Because this will most definitely not work when multiple objects are selected at the same time